Resources/Prototypes/Nyanotrasen/Catalog/Fills/Lockers/epistemics.yml

```
@@ -27,3 +27,5 @@
       - id: ClothingHeadTinfoil
       - id: ClothingHeadCage
         amount: 2
+      - id: LPPRubberStampConfidentional #LPP
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)